chef_master/source/deprecations_chef_platform_methods.rst

```diff
@@ -0,0 +1,112 @@
+=============================================
+Deprecation: Chef::Platform methods (CHEF-13)
+=============================================
+`[edit on GitHub] <https://github.com/chef/chef-web-docs/blob/master/chef_master/source/deprecations_chef_platform_methods.rst>`__
+
+.. tag deprecations_chef_platform_methods
+
+Several methods under `Chef::Platform` that were previously public APIs to control resolution of provider classes were replaced by the dyanmic
```

Comment:
Any inline code blocks in restructuredText need two backticks (instead of one like markdown).